docs/doc_5_best_practices.md

```diff
@@ -0,0 +1,61 @@
+# Best Practices
+
+An number of best practices for Atlas SDK users.
+
+## Using Getters Instead of Direct Field Access
+
+When accessing responses prefer the getter function instead of direct field access.
+
+Instead of `response.Field` use `response.GetField()`.
+
+Using Getters will allow seamless pointer handling and help you to avoid panic errors.
+Additionally SDK provides `Set`, `IsSet` and `Unset` methods for safe field modifications.
+
+## Check for Empty Strings for String Pointers
+
+When model contains an pointer to the `string` 
+we need to be aware that setting that field to the empty string (`""`)
+will send that value to the server.
+
+Instead of direct assignment:
+```go
+test := ""
+requestBody.StringPointerValue = test
+```
+
+Users should always check for empty strings before assigning them:
+```go
+if test != ""   
+    requestBody.StringPointerValue = test
+```
+
+> NOTE: if field is an string type and empty string (`""`) assignment 
+will be valid.
```

Comment:
I would remove this because it reiterates the above point unless I'm misunderstanding 

```suggestion
```